src/Rings/groebner.jl

```diff
@@ -201,7 +207,7 @@ function standard_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_orde
       I.gb[ordering] = IdealGens(GB_dehom_gens, ordering, isGB = true)
     end
   elseif algorithm == :f4
-    groebner_basis_f4(I, complete_reduction=complete_reduction)
+    groebner_basis_f4(I, complete_reduction=true)
```

Comment:
Why is `complete_reduction` being ignored / overridden here? Maybe a comment could be added to explain it?

Reply:
Added a comment on this.